src/bokeh/core/_templates/autoload_request_tag.html

```diff
@@ -32,6 +32,7 @@
     {% for header, value in headers.items() %}
     xhr.setRequestHeader("{{ header }}", "{{ value }}")
     {% endfor %}
+    xhr.withCredentials = {{ with_credentials | lower }};
```

Comment:
I prefer keeping a standard `bool` on the Python side. Since JS uses `true` instead of `True`, we need to lowercase this here. LMK if you want me to do this on the Python side.

Reply:
Isn't it already a bool on the Python side? So should it be?
```suggestion
    xhr.withCredentials = {{ with_credentials | to_json }};
```

Reply:
Yes, bad phrasing. For pure `bool`s `| lower` works as well as it coerces the input into a string first. But `to_json` communicates the intent better.

Reply:
I believe it is `tojson` (no underscore)
```
jinja2.exceptions.TemplateAssertionError: No filter named 'to_json'.
```